### ISSUE TITLE
Option to move dead players to a new channel

### ISSUE THREAD

**Issue by lobstrain** [pedrofracassi/amongcord#22]
I don't know how common this is, but on our server we use a different voice channel for ghosts. It would be nice if the functionality of moving dead players to a different channel was built-in to this bot.

**Comment by iiftekhar** [comment]
yes, it would be great. I think this issue has a similarity with #8

**Comment by ThePhantomOTO** [comment]
The only issue I see with this is the dead player COULD come back to the main chat during discussion and cause issues by talking. I like the fact that the dead can talk while players are doing tasks just fine as the BOT keeps them from being able to interfere with the game for the living.
